mbed_build/_internal/cmake_renderer.py

```diff
@@ -3,41 +3,25 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 import pathlib
+from typing import List
 
 import jinja2
 
 TEMPLATES_DIRECTORY = pathlib.Path("_internal", "templates")
 TEMPLATE_NAME = "CMakeLists.tmpl"
 
 
-def write_cmakelists_file(output_directory: str) -> None:
-    """Writes out the CMakeLists.txt file to the output directory.
-
-    If the intermediate directories to the output directory don't exist,
-    this function will create them.
-
-    This function will overwrite any existing CMakeLists.txt file in the
-    output directory.
-
-    Args:
-        output_directory: path to directory where we want the CMakeLists.txt to go
-    """
-
-    export_directory = pathlib.Path(output_directory)
-    export_directory.mkdir(parents=True, exist_ok=True)
-
-    output_file = export_directory.joinpath("CMakeLists.txt")
-    output_file.write_text(_render_cmakelists_template())
-
-
-def _render_cmakelists_template() -> str:
+def render_cmakelists_template(target_labels: List[str], toolchain_labels: List[str]) -> str:
     """Loads the CMakeLists.txt file template and renders it with the correct details.
 
+    Arguments:
```

Comment:
Is it `Arguments` or `Args`? 🤔 Does it matter?

Reply:
Google style guide says `Args`. I'd suggest we just follow their guidelines for docstrings.

Reply:
👍 Ill change it

Reply:
079dca8